src/index.html

```diff
@@ -20,8 +21,402 @@
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+
+    <main class="main">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__img"
+        />
+
+        <p class="card__product-description">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product-code">Product code: 195434</p>
+
+        <!-- #region stars -->
+        <div class="stars">
+          <div class="stars__rate">
+            <div class="stars__rate-star"></div>
+            <div class="stars__rate-star"></div>
+            <div class="stars__rate-star"></div>
+            <div class="stars__rate-star"></div>
+            <div class="stars__rate-star"></div>
+          </div>
+          <p class="stars__reviews">Reviews: 5</p>
+        </div>
+        <!-- #endregion -->
+
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <button
+          type="button"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </div>
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__img"
+        />
+
+        <p class="card__product-description">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product-code">Product code: 195434</p>
+
+        <!-- #region stars -->
+        <div class="stars">
+          <div class="stars__rate">
+            <div class="stars__rate-star"></div>
+            <div class="stars__rate-star"></div>
+            <div class="stars__rate-star"></div>
+            <div class="stars__rate-star"></div>
+            <div class="stars__rate-star"></div>
+          </div>
+          <p class="stars__reviews">Reviews: 5</p>
+        </div>
+        <!-- #endregion -->
+
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <button
+          type="button"
+          class="card__button"
+        >
+          Buy
+        </button>
+      </div>
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
```

Comment:
Ensure that the image path is correct and that the image exists at the specified location. If this is a placeholder, consider using a more descriptive alt text for accessibility.